cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -176,7 +176,7 @@ func (s *mysqlSyncPointStore) CreateSyncTable(ctx context.Context) error {
 		}
 		return cerror.WrapError(cerror.ErrMySQLTxnError, err)
 	}
-	query := `CREATE TABLE IF NOT EXISTS %s 
+	query := `CREATE TABLE IF NOT EXISTS %s
```

Comment:
What if the table exists? How to migrate it from old schema?

Reply:
PM said there is no need to consider compatibility, since this feature is GA at 6.3.0.